.github/workflows/build-and-test.yml

```diff
@@ -39,7 +48,7 @@ jobs:
 
   publish:
```

Comment:
why do we need a separate stage called `publish`? Why can't the last step be merged into the above stage itself?

Reply:
Merged publish stage to the existing stage